test/terraform_aws_logs_combined_test.go

```diff
@@ -14,23 +13,27 @@ import (
 
 func TestTerraformAwsLogsCombined(t *testing.T) {
 	// Note: do not run this test in t.Parallel() mode.
+	configName := fmt.Sprintf("aws-config-%s", strings.ToLower(random.UniqueId()))
+	expectedConfigLogsBucket := fmt.Sprintf("terratest-%s", configName)
 
 	tempTestFolder := test_structure.CopyTerraformFolderToTemp(t, "../", "examples/combined")
 	testName := fmt.Sprintf("terratest-aws-logs-%s", strings.ToLower(random.UniqueId()))
 	// AWS only supports one configuration recorder per region.
 	// Each test using aws-config will need to specify a different region.
-	awsRegion := "us-east-2"
```

Comment:
@ralren are you talking about this?

Reply:
Now that we added the config names as unique, can we change the regions back to the original values and still have passing tests? 

If so we should revert those changes and update the comment above. 🙂

Reply:
What Misty said.

Reply:
ahhhhhhhh ok ok. makes sense so I retested it and it works with the original regions and it worked. The regions just had to be different from us-west-2